### ISSUE TITLE
Remove all non-GA plugins from the single container

### ISSUE THREAD

**Issue by bmbouter** [pulp/pulp-oci-images#265]
The single container ships plugins that are not GA'd currently. Specifically, the ones below. The expectation from users that I have is that the single container could be run in production so having these non GA plugins there is not aligned with that. We can add them back when they are GA, or later we can make another container that is for more experimental plugins.

```
pulp-cookbook 0.1.0b9
pulp-gem 0.0.1b2
pulp-npm 0.1.0a4
pulp-ostree 2.0.0a6
```

This change would be to remove these from the single container.

**Comment by fao89** [comment]
that's why they are on nightly tag

**Comment by bmbouter** [comment]
Yes but nightly or not, this container is designed to become production ready so why are there non GA plugins in it?

**Comment by mdellweg** [comment]
This nightly tagged container image is built nightly from all the main branches. This will never be considered stable.
This nightly build from GA'd released plugins is called 'latest'.

**Comment by bmbouter** [comment]
That's not what I expect at all since it's the same container name. The tags should identify different points in the release flow of the same set of software, not a different set of software entirely.

If we want a container that has non-GA plugins in it, it should be another container and that can also have a "nightly" for upcoming changes, and "latest" for the released changes. In that case for that other container it would have the same set of software just at different release cycles.

**Comment by mdellweg** [comment]
As long as we think we might eventualy ga those plugins, i see no conflict here. It's something that may eventually become the official pulp single container. Also this CI is kind of a canary for less maintained plugins.

What i really care about is the pulp-cli CI, and none of the removed plugins is used there. So i guess it's ok for me. I just don't see the arguments as compelling. Should we build another tag including as many development versions as we can? Call it `devel` or `unstable`?

**Comment by mikedep333** [comment]
@bmbouter @dkliban and I propose that we put the non-GA plugins in separate container image names, as follows:
- pulp-non-ga-plugins
- pulp-minimal-non-ga-plugins

These are below 30 characters, the limit.
